build/features.test.ts

```diff
@@ -58,7 +58,7 @@ class FeatureFile {
 		return existsSync(this.path);
 	}
 
-	// eslint-disable-next-line n/prefer-global/buffer -- Type only
+	// eslint-disable-next-line @typescript-eslint/ban-types -- Just passing it
 	contents(): Buffer {
```

Comment:
`Buffer` is a `Uint8Array` subclass, so you could just use
```suggestion
	contents(): Uint8Array {
```

here.

Reply:
Buffer has an `includes(string)` method but `Uint8Array` doesn't. Since that's what I'm using it for, I'll keep Buffer for now so I can merge this. Happy to change it if there's a similar method that doesn't require conversions. Maybe later I'll just read it as utf-8 instead